scipy/optimize/_differentialevolution.py

```diff
@@ -588,8 +774,8 @@ def _mutate(self, candidate):
 
         fill_point = self.random_number_generator.randint(0, parameter_count)
 
-        if (self.strategy == 'randtobest1exp'
-                or self.strategy == 'randtobest1bin'):
```

Comment:
The PEP8 bot doesn't care about this (neither do I, I don't think there's a strong preference for either). If you install the pep8 tool version 1.5.1, it should correspond to what's on Travis.